ollama ls crash

### Bug description

When executing `ollama ps` against Podman AI Lab, the following crash happens:
```

panic: runtime error: slice bounds out of range [:12] with length 0

goroutine 1 [running]:
github.com/ollama/ollama/cmd.ListHandler(0xc000246308, {0x7ff6f3f7f7a0, 0x0, 0x7ff6f343a048?})
        C:/a/ollama/ollama/cmd/cmd.go:456 +0x4ec
github.com/spf13/cobra.(*Command).execute(0xc000246308, {0x7ff6f3f7f7a0, 0x0, 0x0})
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x85c
github.com/spf13/cobra.(*Command).ExecuteC(0xc00020c908)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:985
main.main()
        C:/a/ollama/ollama/main.go:12 +0x4d
```

### Operating system

Win11Pro

### Installation Method

None

### Version

next (development version)

### Steps to reproduce

ollama expected the digest for a model to be always present

### Relevant log output

```shell

```

### Additional context

_No response_

--- a/packages/backend/src/managers/apiServer.ts
+++ b/packages/backend/src/managers/apiServer.ts
@@ -55,7 +55,7 @@ function asListModelResponse(model: ModelInfo): ListModelResponse {
   return {
     model: model.id,
     name: model.name,
-    digest: model.sha256,
+    digest: toDigest(model.name, model.sha256),
     size: model.file?.size,
     modified_at: model.file?.creation?.toISOString(),
     details: {},

